tools/code_analyzer/op_dependency.cpp

```diff
@@ -0,0 +1,396 @@
+#include <deque>
+#include <iostream>
+#include <set>
+#include <unordered_map>
+#include <unordered_set>
```

Comment:
I guess it would be nice if the PR description were included inline here, so it would be easier for people to find the comments.